### PR TITLE
Delegation CLI commands

### PR DESCRIPTION
**Pull request by pahor167**
### Description

PR introduces new cli commands related to delegation
delegate
revoke-delegate
max-delegatees-count
update-delegated-amount

### CODE REVIEW

File: packages/cli/src/commands/lockedgold/update-delegated-amount.ts [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+  ]
+
+  async run() {
+    const res = this.parse(UpdateDelegatedAmount)
+    const address: Address = res.flags.from
+    const to: Address = res.flags.to
+
+    this.kit.defaultAccount = address
+
+    await newCheckBuilder(this).isAccount(address).isAccount(to).isAccount(to).runChecks()
```
Comment (aaronmgdr) [inline packages/cli/src/commands/lockedgold/update-delegated-amount.ts]: isAccount(to) is called twice

**Comment by therealharpaljadeja** [comment]
Can we add the following commands too? @pahor167 

- total percent delegated by an account
- list of delegatees for a delegator
- total delegated votes to an address

image

**Comment by pahor167** [comment]
> Can we add the following commands too? @pahor167
> 
> * total percent delegated by an account
> * list of delegatees for a delegator
> * total delegated votes to an address
> 
> image

I added new command `delegate-info` that shows required information

**Review by therealharpaljadeja — APPROVED** [review]
LGTM!

**Review by aaronmgdr — CHANGES_REQUESTED** [review]
For PRs that modify published packages such as CLI  Please add a changeset by running `yarn changeset add`

**Comment by aaronmgdr** [comment]
Isn't CR10 already out? Why is this not merging into master?

**Comment by pahor167** [comment]
> For PRs that modify published packages such as CLI Please add a changeset by running `yarn changeset add`

There doesn't seem to be any changeset in CR10 yet (we will have to wait for it to be merged to master probably)

**Comment by pahor167** [comment]
> Isn't CR10 already out? Why is this not merging into master?

Unfortunately CR10 was not merged into master yet @martinvol is working on that

**Comment by therealharpaljadeja** [comment]
@pahor167 CR10 is merged!

**Comment by pahor167** [comment]
I had some issues rebasing to master so I published new PR https://github.com/celo-org/celo-monorepo/pull/10702

**Pull request closed without merging**